### PR TITLE
renice: group to process group

### PR DESCRIPTION
**Pull request by FraYoshi**
Actually, the "g" flag renices the Parent Process ID, not the group of the user as it seems like reading the page. 

----

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/renice.md [hunk 1]
```diff
@@ -10,6 +10,6 @@
 
 `renice -n {{niceness_value}} -u {{user}}`
 
-- Change priority of all processes that belongs to a group:
+- Change priority of all processes that belongs to a group of processes (PPID):
 
-`renice -n {{niceness_value}} -g {{group}}`
+`renice -n {{niceness_value}} -g {{PPID}}`
```
[the remark this answers is not in the document]
Comment (mfrw) [reply pages/common/renice.md]: @FraYoshi Can you please change `-g` to `--pgrp` and `PPID` to `process_group`.
I have already suggested it.

**Comment by agnivade** [comment]
Great fix ! Thanks @FraYoshi for the PR. And thanks @mfrw for investigating deeper into it !

File: pages/common/renice.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/renice.md]: This is because we prefer the long-form options in tldr-pages to aid clarity :-)

**Comment by FraYoshi** [comment]
the suggestion should now be applied : )
sorry, tried the new GitHub beta feature that didn't work out completely ^^

**Comment by mfrw** [comment]
@FraYoshi Thanks for sticking with our review process.
👍

**Pull request merged**